pkg/controller/seed-controller-manager/mla/org_grafana_controller_test.go

```diff
@@ -62,7 +62,7 @@ func newTestOrgGrafanaReconciler(t *testing.T, objects []ctrlruntimeclient.Objec
 
 	grafanaClient := grafanasdk.NewClient(ts.URL, "admin:admin", ts.Client())
 
-	orgGrafanaController := newOrgGrafanaController(dynamicClient, kubermaticlog.Logger, grafanaClient)
+	orgGrafanaController := newOrgGrafanaController(dynamicClient, kubermaticlog.Logger, "mla", grafanaClient)
```

Comment:
Is it easy and possible to add some unit tests for dashboards?

Reply:
yep, added test